code/game/sound.dm

```diff
@@ -222,6 +222,8 @@ distance_multiplier - Can be used to multiply the distance at which the sound is
 				soundin = pick('sound/effects/rustle1.ogg','sound/effects/rustle2.ogg','sound/effects/rustle3.ogg','sound/effects/rustle4.ogg','sound/effects/rustle5.ogg')
 			if (SFX_BODYFALL)
 				soundin = pick('sound/effects/bodyfall1.ogg','sound/effects/bodyfall2.ogg','sound/effects/bodyfall3.ogg','sound/effects/bodyfall4.ogg')
+			if (SFX_CLOCKFALL)
+				soundin = 'sound/creatures/clockwork_golem_dead.ogg'
```

Comment:
theres no reason to use the string defines for just 1 file, you can use that file directly